cclib/method/volume.py

```diff
@@ -286,7 +286,7 @@ def electrondensity_spin(ccdata, volume, mocoeffslist):
                         data[i, j, :] = tmp
                 data *= mocoeff[bs]
                 wavefn += data
-            density.data += wavefn ** 2
+            density.data += wavefn ** 2 # lastly put them into second power

```

Comment:
Is there a reason for this comment?

Reply:
This looks like one of the comments that I've left in the earlier PR while reading through the code and forgot to remove. It looks like somehow this comment found its way back in :0 I've removed this. Thanks for letting me know.